### PR TITLE
Fix the issue with empty comments and the style suggestions of Andrey.

### PR DESCRIPTION
**Pull request by ancamaria99**
Also added a file that was missing before : DeleteCommentsServlet.java

### CODE REVIEW

File: ancar/portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
```
Comment (stofee) [inline ancar/portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java:1]: this file is not mentiond in the desc...

Comment (ancamaria99) [reply ancar/portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java:1]: So sorry! Apparently it wasn't tracked and I didn't notice and added it now. 
Room for improvement :(

Comment (ancamaria99) [reply ancar/portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java:1]: It was from the previous pull request

Comment (stofee) [reply ancar/portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java:1]: so do you want to revert it, or add it to the description? alternatively create a ranch with only this, have an other pr, and rebase this pr, so it does not show it anymore, but the file is in the master.. ;)

Comment (ancamaria99) [reply ancar/portfolio/src/main/java/com/google/sps/servlets/DeleteCommentsServlet.java:1]: I want to add it to the description.

**Pull request merged**